omnibus/files/private-chef-cookbooks/private-chef/libraries/elasticsearch.rb

```diff
@@ -1,8 +1,12 @@
+# frozen_string_literal: true
+
 # A class that knows about Elasticsearch configuration and usage.
 class Elasticsearch
   KB = 1024
   MB = KB * KB
   GB = MB * KB
+  MINMEMORY = 1 * KB
```

Comment:
The heap size is in MB. Using `KB` here might give the impression that the unit size is KB.

Also, `MIN_HEAP_SIZE` and `MAX_HEAP_SIZE` might be more descriptive,